src/main/scala/zio/keeper/Cluster.scala

```diff
@@ -0,0 +1,46 @@
+package zio.keeper
+
+import java.net.InetAddress
+
+import zio.{ IO, ZIO }
+import zio.stream.Stream
+import zio.keeper.Cluster._
+
+trait Cluster {
+  final def join: ZIO[Credentials with Discovery with Transport with Metadata, Error, Handle] = ???
+}
+
+object Cluster {
+  // TODO: properly represent (see current 'develop' branch)
+  type Path[_, _]
+  type Type[_]
+
+  object Type {
+    def apply[A](implicit instance: Type[A]): Type[A] = instance
```

Comment:
I'd remove this and Path in the initial implementation.

Reply:
Done.